addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -126,9 +126,9 @@ public void run() {
                 }
             } catch (Throwable t) {
                 if (t instanceof Exception) {
-                    logger.error(((Exception) t).getMessage());
+                    logger.error("{}", t.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace